core-bundle/contao/library/Contao/DcaLoader.php

```diff
@@ -159,7 +159,7 @@ private function addDefaultLabels()
 
 			foreach ($GLOBALS['TL_DCA'][$this->strTable]['list'][$key] as $k=>&$v)
 			{
-				if (isset($v['label']))
+				if (array_key_exists('label', $v))
```

Comment:
As this point the reference to the `TL_LANG` array already exists. But if the language file was not loaded yet, the value of the reference is still `NULL`.